packages/pieces/apps/src/lib/trello/actions/create-card.ts

```diff
@@ -0,0 +1,53 @@
+import { createAction, httpClient, HttpRequest, HttpMethod, Property } from '@activepieces/framework';
+import { trelloCommon } from '../common';
+import { TrelloCard } from '../common/props/card';
+
+export const createCard = createAction({
+	name: 'create_card',
+    displayName:'Create Card',
+    description: 'Create a new card in Trello',
+	props: {
+        authentication: trelloCommon.authentication,
+        token: trelloCommon.token,

```

Comment:
We can replace authentication and token, with basic authentication property It can hold two secret text.

I understand it's not exactly Basic Authentication, but you can consider api key as username and token as password for time being, check basic auth property I just updated the docs

https://github.com/activepieces/activepieces/blob/main/docs/contributing/building-pieces/reference.mdx